src/sched/futex.c

```diff
@@ -45,6 +45,11 @@ static uint32_t to_futex_key(int* uaddr)
     return (uint32_t)uaddr;
 }
 
+void futex_init()
```

Comment:
This is an lthread futex call, but declared in the pthread futex header.

Reply:
Ah, good point. I put it there because `futex_tick()` is already defined there. I presume that should be moved as well?

Reply:
There are also other changes to the file, which I don't quite understand.

Reply:
Yes, I think they should both live in lthread.h.

Reply:
Moved and reverted futex.h to the host-musl version.